plugins/clipboard/dialogs/paste.js

```diff
@@ -240,7 +240,7 @@ CKEDITOR.dialog.add( 'paste', function( editor ) {
 /**
  * Internal event to pass paste dialog's data to the listeners.
  *
- * This event was not available in 4.7.0-4.8.0 versions.
+ * This event was not available in 4.7.0-4.9.0 versions.
```

Comment:
This should remain unchanged. This event was not available till 4.8.0 (inclusive). Starting with version 4.9.0 it is available again.

Reply:
Yes, my bad, now I see it makes perfect sense with `4.7.0-4.8.0` 👍